compiler/lib/src/main/scala/codegen/CppWriter/ComponentCppWriter/TestWriter/ComponentTesterBaseWriter.scala

```diff
@@ -627,6 +630,71 @@ case class ComponentTesterBaseWriter(
     )
   }
 
+
+  private def getDispatcherFunctions: List[CppDoc.Class.Member] = {
+    val oneBody = intersperseBlankLines(
+      List(
+        lines(
+          """|    // Dispatch one message returning status
+             |    return component.doDispatch();
+             |"""
+        ),
+      )
+    )
+    val allBody = intersperseBlankLines(
+      List(
```

Comment:
I think we don't need the `intersperseBlankLines` and we don't need the leading spaces in the body lines.